CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 ## v0.4.2
 - Added an `Init` struct, which can help with initial routing (Breaking)
+- Changed `Tag::Custom` to `Tag::from` (Breaking)
```

Comment:
I've updated `from` function to accept more input types, no other changes, i.e. no breaking changes.